lib/parser.js

```diff
@@ -134,23 +134,24 @@ function parseFromUrl(url, fetchOptions = {}, options) {
 async function dereference(parsedJSON, initialFormat, asyncapiYAMLorJSON, options) {
   try {
     return await refParser.dereference(options.path, parsedJSON, {
+      continueOnError: true,
       parse: options.parse,
       resolve: options.resolve,
       dereference: options.dereference,
     });
   } catch (err) {
     throw new ParserError({
       type: 'dereference-error',
-      title: err.message,
+      title: err.errors[0].message,
       parsedJSON,
-      refs: findRefs(parsedJSON, err.originalPath || err.path, options.path, initialFormat, asyncapiYAMLorJSON),
+      refs: findRefs(err.errors, initialFormat, asyncapiYAMLorJSON),
```

Comment:
no need to pass `parsedJSON` as we no longer need to traverse through it to find a location where the error happened